lib/gui/etcher.js

```diff
@@ -40,10 +40,10 @@ electron.app.on('ready', () => {
     height: 380,
     useContentSize: true,
     show: false,
-    resizable: false,
+    resizable: Boolean(process.env.ETCHER_FULLSCREEN),
```

Comment:
It still shouldn't be resizable if fullscreen I think

Reply:
Seems to be required for some reason along with fullscreening. Although weirdly enough you can remove `fullscreenable` and it's still fullscreen.